README.md

```diff
@@ -1,2 +1,29 @@
 # devtools
 Development tools for SlamData
+
+## Instructions
+How to get up and running with `merge`.
+
+0. Before using merge you must first approve the pull request you have been assinged. 
```

Comment:
This might be better suited for a general merge process document. Also, it's not required for the merge script to work and is per PR rather than initial setup.